crates/subspace-node/src/commands/domain_key.rs

```diff
@@ -94,8 +92,6 @@ pub struct InsertDomainKeyOptions {
 }
 
 pub fn insert_domain_key(options: InsertDomainKeyOptions) -> Result<(), Error> {
-    init_logger();
```

Comment:
Looks like this init was accidentally deleted?

Reply:
Yep same for wipe